src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -329,6 +333,11 @@ export default class NameTranslation extends Mixins(CommonMixin) {
   // Watchers
   @Watch('nameTranslations', { deep: true, immediate: true })
   private onNameTranslationsPropValueChanged (): void {
+    if (this.isSummaryMode) {
+      this.nameTranslations.forEach(function (translation) {
+        translation.name = translation.name.toUpperCase()
```

Comment:
Does this make the name appear as uppercase as the user enters it?

Can you use what I've started in my PR? https://github.com/bcgov/business-edit-ui/pull/142

Reply:
Did we decide that we would show uppercase but store in whatever case the user typed in?

Reply:
I am now using your "toUpperCase" function, so it will be in uppercase in both cases